start.php

```diff
@@ -15,7 +15,7 @@
 	 *
 	 * @var string
 	 */
-	$this_sdk_version = '2.5.11';
+	$this_sdk_version = '2.5.12';
```

Comment:
@fajardoleo Rebase this branch please and bump the version by our convention. `2.5.12.1` if I am not mistaken?